accepted/platform-intrinsics.md

```diff
@@ -14,30 +14,30 @@ consistent, imposes a performance penalty unacceptable to implementors seeking
 maximum app/service throughput.  To enable this last bit of performance
```

Comment:
nit: in original, not modified text, Intel is sometimes used without first uppercase letter